### PR TITLE
Assault Pods & Fixes

### PR DESCRIPTION
**Pull request by tigercat2000**
@Fox-McCloud @TheDZD
Fixes:
 - Lighting system bandaid: Lighting overlays can't go below 0 lum ever
 - -tg- shuttles now use changeTurf() and forceMove() appropriately
 - forceMove() actually bothers to update lighting now
 - Fixed teleportlocs including some prohibited areas. 

Misc changes:
 - Moved the emergency shuttle stuff to the modules/shuttles/ folder
 - Moved the cargo shuttle stuff to the modules/shuttles/ folder

Features:
 - Added nukeops assault-pod. Nuke ops may purchase a destination setter
   for 30 telecrystals.
   The assault pod is a shuttle, equipped with 8 turrets that use
   weakbullet3. When an area is selected, a destination will be randomly
   placed in one of the turfs in that area. The pod will gib anyone
   standing in the area where it lands, and overwrite any turfs.

## Screenshots!
assault pod
nukie base

:cl:
rscadd: Nukeops now have an assault pod, they can use it for 30 TC. Loud and mean.
bugfix: Lighting overlays can no longer go below 0 lum_r/g/b
bugfix: Shuttles will work with lighting better.
bugfix: Wizards can no longer teleport to prohibited areas such as Central Command.
/:cl:

### CODE REVIEW

**Comment by johannhawk** [comment]
oh no! the pAI is gone now!

anyways, this gives nukies a incentive to send in one covert guy, and then summoning the pod once the covert guy finds who has the disk!

**Comment by Mijhead** [comment]
:-1: return the nukie pAI!

**Comment by Magikarpet** [comment]
doesn't tg keep the pai on the current location of the beer, on the table?

**Comment by Spacemanspark** [comment]
:+1: But please add the pAI back.
Been wanting to see this for a bit now.

**Comment by monster860** [comment]
You forgot fix, @Fox-McCloud

**Comment by monster860** [comment]
Anyways, if I had this, I'd just teleport right into the armory and take all the guns.

**Comment by tigercat2000** [comment]
@monster860 problem with that is when the guns all get destroyed :P

**Comment by monster860** [comment]
No, I'd teleport into ARMORY, use emag to break into SECURE ARMORY. They are different areas. I could also go into the warden's office if I was that afraid

**Comment by Tastyfish** [comment]
I'm tempted to say it should _only_ be functional if war is declared, since that would remove any concerns about them dropping into the armory, etc unannounced. Otherwise, most of the areas already have backups (eg bridge) or would kill the ops (eg engine containment).

**Comment by Fox-McCloud** [comment]
I'd like to request two mapping changes, to keep this in line with TG and also to retain existing functionality.

- a toy nuke on the table above the windoor
- a pAI on the table to the left of the pizza box

File: code/game/machinery/turrets.dm [hunk 1]
```diff
@@ -566,4 +570,10 @@
...
 	name = "mounted grenade launcher (40mm)"
 	desc = "Syndicate 40mm grenade launcher defense turret. If you've had this much time to look at it, you're probably already dead."
-	bullet_type = /obj/item/projectile/bullet/a40mm
\ No newline at end of file
+	bullet_type = /obj/item/projectile/bullet/a40mm
+
+/obj/machinery/gun_turret/assault_pod
+	name = "machine gun turret (4.6x30mm)"
+	desc = "Syndicate exterior defense turret chambered for 4.6x30mm rounds. Designed to be fitted to assault pods, it uses low calliber bullets to save space."
+	health = 100
```
Comment (Fox-McCloud) [inline code/game/machinery/turrets.dm]: these have a health of 40, on TG.

Comment (tigercat2000) [inline code/game/machinery/turrets.dm]: @Fox-McCloud and turrets have a default health of 80 on -tg-
Our syndicate-ship has turrets with 200 health. Same ratio.

**Comment by tigercat2000** [comment]
readded the toys n shit, see OP

**Pull request merged**